CMakeLists.txt

```diff
@@ -481,11 +481,10 @@ ELSE()
 ENDIF(WIN32)
 
 #this is the product (a library)
-if(${build_as_dynamic})
-    add_library(aziotsharedutil SHARED ./src/aziotsharedutil.def ${source_c_files} ${source_h_files})
-else()
-    add_library(aziotsharedutil ${source_c_files} ${source_h_files})
-endif()
+add_library(aziotsharedutil ${source_c_files} ${source_h_files})
+
+#make sure we can link as dll/so
+add_library(aziotsharedutil_dll SHARED ./src/aziotsharedutil.def ${source_c_files} ${source_h_files})
```

Comment:
Sure, done.

---
In reply to: [118510420](https://github.com/Azure/azure-c-shared-utility/pull/78#discussion_r118510420) [](ancestors = 118510420)